mlperf_logging/result_summarizer/result_summarizer.py

```diff
@@ -521,23 +521,35 @@ def _compute_power_node(loglines, time_to_train):
     power_stop = 0
     agg_power = 0
     conversion_eff = 1.0
+    power_stop_found = False
+    power_start_found = False
+    loglines.sort(key=lambda x: x.timestamp)
     for logline in loglines:
         if logline.key == "power_measurement_start":
             power_start = logline.timestamp
             prev_timestamp = logline.timestamp
-        if logline.key == "power_reading":
-            agg_power += (logline.value['value'] * (logline.timestamp - prev_timestamp))
+            power_start_found = True
```

Comment:
if the power-start line is not found in the log, should we raise an error or log a warning? i guess it might cause an inaccurate calculation if we assumed 0 start time.

Reply:
Yes, we should rase an error. Just updated the code

Reply:
https://github.com/mlcommons/logging/blob/640c988d1ee60123414f05a11fef91415ecdbe1e/mlperf_logging/result_summarizer/result_summarizer.py#L546-L548